documentcloud/documents/admin.py

```diff
@@ -56,3 +58,30 @@ def save_model(self, request, obj, form, change):
 
     def delete_model(self, request, obj):
         obj.destroy()
+
+
+@admin.register(Entity)
+class EntityAdmin(admin.ModelAdmin):
+    """Entity Admin"""
+
+    list_display = ("name", "kind", "mid", "description", "wikipedia_url")
+    list_filter = ("kind", "description")
+    search_fields = ("name", "kind", "mid", "description", "wikipedia_url")
+    show_full_result_count = False
+    paginator = LargeTablePaginator
+    ordering = ("pk",)
+    fields = ("name", "kind", "mid", "description", "wikipedia_url")
+
+    readonly_fields = ()
+
+    @transaction.atomic
+    def save_model(self, request, obj, form, change):
```

Comment:
`solr_index` indexes a document into the Solr search backend - there is no reason to index entity's like this.  We can go over how Solr works in more detail at some point.

Reply:
OK, it would be good to know! I was just guessing that people would want to search for entities.

Reply:
This method can be removed for now